hs_core/templates/pages/genericresource.html

```diff
@@ -329,9 +329,9 @@
             {% elif rights.statement == "This resource is shared under the Creative Commons Attribution-NoCommercial-NoDerivs CC BY-NC-ND." %}
                 <img class="cc-image" src="{{ STATIC_URL }}img/cc-badges/CC-BY-NC-ND.png" alt="CC-BY-NC-ND"/>
             {% endif %}
-        {% endif %}
-    {% else %}
-        <div class="row">
+            {% endif %}
+            {% else %}
+            <div class="row">
```

Comment:
@mauriel133 I see you removed the indentation for marking each conditional block. I prefer to have the block indentation which makes code more structured and easier to read, but since genericresource template file is so big now with deep indentation levels with complicated logic, I will defer to you whether you want the block indentation or not in this case since you are the person who works on this template file the most and will do the refactoring for this template file when the time comes.